<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Code_Audit</class>
 <widget class="QWidget" name="Code_Audit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>601</width>
     <height>231</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; font-size:12pt; color:#000000;&quot;&gt;&amp;lt;?php&lt;br /&gt;    $con=mysqli_connect(&amp;quot;localhost&amp;quot;,&amp;quot;root&amp;quot;,&amp;quot;XFAICL1314&amp;quot;,&amp;quot;dvwa&amp;quot;);&lt;br /&gt;    if(mysqli_connect_error())&lt;br /&gt;    {&lt;br /&gt;        echo &amp;quot;&lt;/span&gt;&lt;span style=&quot; font-family:'宋体','monospace'; font-size:12pt; color:#000000;&quot;&gt;连接失败&lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; font-size:12pt; color:#000000;&quot;&gt;:&amp;quot; .mysqli_connect_error();&lt;br /&gt;    }&lt;br /&gt;    $id=$_GET['id'];&lt;br /&gt;    $result=mysqli_query($con,&amp;quot;select * from users where `user_id`=&amp;quot;.$id);&lt;br /&gt;    $row=mysqli_fetch_array($result);&lt;br /&gt;    echo $row['user'] . &amp;quot;:&amp;quot; . $row['password'];&lt;br /&gt;    echo &amp;quot;&amp;lt;br&amp;gt;&amp;quot;;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>0</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string></string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
